tests/framework/common.go

```diff
@@ -88,6 +88,14 @@ func OSMDescribe(name string, opts OSMDescribeInfo, body func()) bool {
 	return Describe(fmt.Sprintf("%s %s", opts, name), body)
 }
 
+const (
+	// AppPort is the port to be used for server and client applications in e2es
```

Comment:
A note here that port 80 is reserved by OpenShift wouldn't hurt.

Reply:
```suggestion
	// ... is the default port on which the server apps listen for connections from client apps.
	// Note: Port 80 should not be used because it does not work on OpenShift.
```